src/mod_filtering.c

```diff
@@ -687,16 +664,9 @@ void nmdi_filtering_free(struct filtering_ctx **fp)
         av_freep(&ctx->window_func_lut);
         av_freep(&ctx->rdft_data[0]);
         av_freep(&ctx->rdft_data[1]);
-#if LIBAVUTIL_VERSION_INT >= AV_VERSION_INT(57, 19, 100)
         if (ctx->avtx) {
             av_tx_uninit(&ctx->avtx);
         }
-#else
```

Comment:
Note: I'm under the impression that the condition is unnecessary

Reply:
Removed in a separate commit.